tools/pythonpkg/tests/fast/relational_api/test_rapi_query.py

```diff
@@ -1,5 +1,11 @@
 import duckdb
 import pytest
+import platform
+
+pytestmark = pytest.mark.skipif(
+    platform.system() == "Emscripten",
+    reason="Not supported on Emscripten",
+)
```

Comment:
Some weird stuff happening here, I get a stack overflow when running some of the tests in this file.